modules/hcp-ibmz-update-reg-ca.adoc

```diff
@@ -0,0 +1,54 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-update-registry-ca-private-registry_{context}"]
+= Update Registry Certificate Authority with Private Registry
```

Comment:
Change the heading to "Update Registry Certificate Authority in AgentServiceConfig with Mirror Registry "

Reply:
Fixed